<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Xanadu Animation Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,0">
    <item>
     <widget class="QOpenGLWidget" name="viewer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="verticalWidget" native="true">
      <layout class="QVBoxLayout" name="sidebar">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>260</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="tab_details">
          <attribute name="title">
           <string>Details</string>
          </attribute>
          <widget class="QGroupBox" name="sceneDetails">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>256</width>
             <height>74</height>
            </rect>
           </property>
           <property name="title">
            <string>Scene Details</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="fileLabel">
              <property name="text">
               <string>File:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="fileValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="versionLabel">
              <property name="text">
               <string>Version:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="versionValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QListWidget" name="nodeList">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>241</width>
             <height>241</height>
            </rect>
           </property>
          </widget>
          <widget class="QGroupBox" name="nodeDetails">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>340</y>
             <width>77</width>
             <height>118</height>
            </rect>
           </property>
           <property name="title">
            <string>Node Details</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="nodeFlagsLabel">
              <property name="text">
               <string>Flags:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="nodeFlagsValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="vertexCountLabel">
              <property name="text">
               <string>Vertex #:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="vertexCountValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="faceCountLabel">
              <property name="text">
               <string>Face #</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="faceCountValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="childCountLabel">
              <property name="text">
               <string>Child #</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="childCountValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_animations">
          <attribute name="title">
           <string>Animations</string>
          </attribute>
          <widget class="QListWidget" name="animList">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>241</width>
             <height>192</height>
            </rect>
           </property>
          </widget>
          <widget class="QListWidget" name="animRangeList">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>210</y>
             <width>241</width>
             <height>131</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="animNameValue">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>350</y>
             <width>231</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="animArgsValue">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>380</y>
             <width>241</width>
             <height>51</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="animStartlabel">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>460</y>
             <width>49</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Start:</string>
           </property>
          </widget>
          <widget class="QLabel" name="animEndLabel">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>460</y>
             <width>49</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>End:</string>
           </property>
          </widget>
          <widget class="QLabel" name="animRepeatLabel">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>480</y>
             <width>49</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Repeat:</string>
           </property>
          </widget>
          <widget class="QLabel" name="animUnk1Label">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>500</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Unk1:</string>
           </property>
          </widget>
          <widget class="QLabel" name="animUnk2Label">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>500</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Unk2:</string>
           </property>
          </widget>
          <widget class="QLabel" name="animStartValue">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>460</y>
             <width>49</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="animEndValue">
           <property name="geometry">
            <rect>
             <x>188</x>
             <y>460</y>
             <width>71</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="animRepeatValue">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>480</y>
             <width>49</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="animUnk1Value">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>500</y>
             <width>71</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="animUnk2Value">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>500</y>
             <width>91</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="animRangeLabel">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>440</y>
             <width>121</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Animation Range:</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_events">
          <attribute name="title">
           <string>Events</string>
          </attribute>
          <widget class="QListWidget" name="eventList">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>241</width>
             <height>192</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="eventTypeValue">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>210</y>
             <width>49</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="eventTypeName">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>210</y>
             <width>181</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="eventName1Label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>240</y>
             <width>49</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Name 1:</string>
           </property>
          </widget>
          <widget class="QLabel" name="eventName2Label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>260</y>
             <width>49</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Name 2:</string>
           </property>
          </widget>
          <widget class="QLabel" name="eventFrameLabel">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>280</y>
             <width>49</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Frame:</string>
           </property>
          </widget>
          <widget class="QLabel" name="eventIslongLabel">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>300</y>
             <width>49</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Is Long:</string>
           </property>
          </widget>
          <widget class="QLabel" name="eventHeadArgsLabel">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>320</y>
             <width>61</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Head args:</string>
           </property>
          </widget>
          <widget class="QLabel" name="eventTailArgsLabel">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>400</y>
             <width>61</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Tail args:</string>
           </property>
          </widget>
          <widget class="QLabel" name="eventHeadArgsValue">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>340</y>
             <width>211</width>
             <height>61</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="eventTailArgsValue">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>420</y>
             <width>211</width>
             <height>61</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="eventName2Value">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>260</y>
             <width>171</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="eventFrameValue">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>280</y>
             <width>171</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="eventName1Value">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>240</y>
             <width>171</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="eventIslongValue">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>300</y>
             <width>171</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="eventUnknownLabel">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>500</y>
             <width>49</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Unkown</string>
           </property>
          </widget>
          <widget class="QLabel" name="eventUnknownValue">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>500</y>
             <width>161</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="recentMenu">
     <property name="title">
      <string>Recent Files</string>
     </property>
    </widget>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="recentMenu"/>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="fileMenu"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
